### PR TITLE
Feature/signals on chart

### PR DESCRIPTION
**Pull request by GarageInc**
**Summary**

1. Show/create/delete price signals on chart for selected slug
2. Figma:
image

**Screenshots**
image
image

### CODE REVIEW

File: src/assets/signals/buttons/pointer.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg width="6" height="10" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M5.564 4.622a.5.5 0 0 1 0 .756L.827 9.48A.5.5 0 0 1 0 9.102V.898A.5.5 0 0 1 .827.52l4.737 4.102z" fill="#2F354D"/></svg>
```
Comment (haritonasty) [inline src/assets/signals/buttons/pointer.svg]: that should be in our storybook. Ask Anastasya about actual icons. She is working on it right now

File: src/ducks/SANCharts/Charts.js [hunk 2]
```diff
@@ -260,10 +323,29 @@ class Charts extends React.Component {
       yValue: this.props.chartData[activeTooltipIndex][
         tooltipMetric.dataKey || tooltipMetric.key
       ],
-      hovered: true
+      hovered: true,
+      newSignalData: priceUsd
+        ? {
+          priceUsd: priceUsd,
```
Comment (haritonasty) [inline src/ducks/SANCharts/Charts.js]: `priceUsd: priceUsd` -> `priceUsd`

File: src/ducks/SANCharts/components/newSignalLine/SignalLine.module.scss [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+.tooltip {
+  padding: 2px 8px;
+  background: var(--rhino);
+  color: var(--casper);
+  position: relative;
+  left: 8px;
+  bottom: 5px;
+  border-radius: $border-radius;
+  display: table;
```
Comment (haritonasty) [inline src/ducks/SANCharts/components/newSignalLine/SignalLine.module.scss]: `display: table` why?)

File: src/assets/signals/buttons/pointer.svg (hunk 1 above)
Comment (GarageInc) [reply src/assets/signals/buttons/pointer.svg]: Yes, it is on storybook. But we can add background image to svg circle only as URL. In our storybook it is using as icon(not url for direct access). @haritonasty

File: src/ducks/SANCharts/components/newSignalLine/SignalLine.module.scss (hunk 3 above)
Comment (GarageInc) [reply src/ducks/SANCharts/components/newSignalLine/SignalLine.module.scss]: Container has full width, but child needs to fill only available content with text. Display:table solve this problem. flex/block/etc filled 100% of container.

**Pull request merged**